adapter/internal/oasparser/envoyconf/envoyconf_internal_test.go

```diff
@@ -101,7 +91,7 @@ func TestCreateRoute(t *testing.T) {
 					MaxProgramSize: nil,
 				},
 			},
-			Regex: "^/xWso2BasePath/resourcePath(/{0,1})",
+			Regex: "^/xWso2BasePath/resourcePath[/]{0,1}$",
```

Comment:
Remove the $

Reply:
Fixed with https://github.com/wso2/product-microgateway/pull/2983/commits/be675ab2c0ab3a0fc2235fbdb7d2af0accb89767